### PR TITLE
Big Reformat/Organization/Cleanup

### PR DESCRIPTION
**Pull request by stuntguy3000**
This is a big PR.

Code standard is important, so, lets keep it to one. This was done automagically via Intellij. It uses the same settings that I have done in previous PRs. 

This helps to maintain a coding standard, as it bugged me.

This PR features
- Unused imports removed
- Organization of imports, methods and variables
- Proper spacing between various elements (e.g. if statements, no lack-of-braces if statements)
- No tab spacing - Pet peeve

_Personally, the rearrange feature is epic_

### CODE REVIEW

File: src/main/java/com/dsh105/nexus/Nexus.java [hunk 1]
```diff
@@ -36,20 +38,16 @@
...
 
 public class Nexus extends PircBotX {
 
-    private static Nexus INSTANCE;
-
     public static Logger LOGGER = Logger.getLogger(Nexus.class.getName());
     public static PrettyTime PRETTY_TIME = new PrettyTime();
-
+    public static String CONFIG_FILE_NAME = "options.yml";
+    private static Nexus INSTANCE;
```
Comment (DSH105) [inline src/main/java/com/dsh105/nexus/Nexus.java:48]: What was wrong with the positioning of these two? :'(

File: src/main/java/com/dsh105/nexus/command/module/general/RandomNumberCommand.java [hunk 2]
```diff
@@ -17,9 +17,9 @@
         help = "Generate a random number from Random.org",
         extendedHelp =
                 {"{b}{p}{c}{/b} <max> - Chooses a number from 1 to <max>.",
```
Comment (DSH105) [inline src/main/java/com/dsh105/nexus/command/module/general/RandomNumberCommand.java]: Possibly separate this one from the `{` so they're all even?

File: src/main/java/com/dsh105/nexus/hook/github/GitHubIssue.java [hunk 3]
```diff
@@ -22,39 +22,29 @@
 
 public class GitHubIssue {
 
+    protected GitHubRepo repo;
+    protected GitHubUser reportedBy;
     @SerializedName("url")
     private String apiUrl;
-
```
Comment (DSH105) [inline src/main/java/com/dsh105/nexus/hook/github/GitHubIssue.java:27]: But spaces are prettier :c

File: src/main/java/com/dsh105/nexus/Nexus.java (hunk 1 above)
Comment (stuntguy3000) [inline src/main/java/com/dsh105/nexus/Nexus.java:48]: Im really not sure. Its just how intellij does its stufficles with sorting. I assume it is about separation between statics and non statics. (Segregation at its finest)

File: src/main/java/com/dsh105/nexus/command/module/general/RandomNumberCommand.java (hunk 2 above)
Comment (stuntguy3000) [inline src/main/java/com/dsh105/nexus/command/module/general/RandomNumberCommand.java]: What you mean?

File: src/main/java/com/dsh105/nexus/hook/github/GitHubIssue.java (hunk 3 above)
Comment (stuntguy3000) [inline src/main/java/com/dsh105/nexus/hook/github/GitHubIssue.java:27]: lies

File: src/main/java/com/dsh105/nexus/command/module/general/RandomNumberCommand.java (hunk 2 above)
Comment (DSH105) [inline src/main/java/com/dsh105/nexus/command/module/general/RandomNumberCommand.java]: https://github.com/DSH105/Nexus/blob/master/src/main/java/com/dsh105/nexus/command/module/admin/ChannelCommand.java#L35-L38

Not much of an issue. Some of the other command classes probably need some formatting improvements in this area.

File: src/main/java/com/dsh105/nexus/hook/github/GitHubIssue.java (hunk 3 above)
Comment (DSH105) [inline src/main/java/com/dsh105/nexus/hook/github/GitHubIssue.java:27]: nono. no lies. cake is a lie

File: src/main/java/com/dsh105/nexus/command/module/minecraft/MinecraftUserCommand.java [hunk 4]
```diff
@@ -17,7 +16,8 @@
         aliases = {"minecraftuser"},
         needsChannel = false,
         help = "View information about a specific Minecraft username.",
-        extendedHelp = {"{b}{p}{c}{/b} <usernane>- View information about the Minecraft username."})
+        extendedHelp = {"{b}{p}{c}{/b} <usernane>- View information about the Minecraft username."
```
Comment (DSH105) [inline src/main/java/com/dsh105/nexus/command/module/minecraft/MinecraftUserCommand.java]: ^w @stuntguy3000 for formatting

**Pull request merged**